querydsl-maven-plugin/src/main/java/com/querydsl/maven/AbstractMetaDataExportMojo.java

```diff
@@ -33,6 +33,9 @@
 import com.querydsl.sql.codegen.DefaultNamingStrategy;
 import com.querydsl.sql.codegen.MetaDataExporter;
 import com.querydsl.sql.codegen.NamingStrategy;
+import com.querydsl.sql.codegen.support.NumericMapping;
+import com.querydsl.sql.codegen.support.RenameMapping;
+import com.querydsl.sql.codegen.support.TypeMapping;
```

Comment:
The *Mapping classes have been moved to querydsl-sql-codegen, that's why they are now imported.